apps/admin/src/test/setup.ts

```
@@ -0,0 +1,67 @@
+            isIntersecting,
+            target,
+          }) as IntersectionObserverEntry,
+      );
+      this.callback(entries, this);
+    }
+  }
+
+  globalThis.IntersectionObserver = IntersectionObserverMock;
+  (globalThis as typeof globalThis & { __triggerIntersection?: () => void }).__triggerIntersection =
+    () => {
+      IntersectionObserverMock.instances.forEach((observer) => observer.trigger(true));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)